### PR TITLE
added post requests

### PR DESCRIPTION
**Pull request by JokubasD**

### CODE REVIEW

File: src/app/application-dialog/application-dialog.component.ts [hunk 1]
```diff
@@ -97,12 +97,41 @@ export class ApplicationDialogComponent implements OnInit {
...
+      error: (err) => {
+        console.log(err);
+      }
+    })
   }
 
   saveLoanDetails(): void {
     this.applicationData.realEstatePrice = this.loanDetailsForm.value.realEstatePrice;
-//update other fields
+    this.applicationData.downPayment = this.loanDetailsForm.value.downPayment;
```
Comment (kyskiz) [inline src/app/application-dialog/application-dialog.component.ts:113]: Looks repetitive, update to
[‘key1’, ‘key2’, …].forEach((key) => { this.applicationData[key] = this.loanDetailsForm.value[key]});
for style points 😎

Comment (swoogie) [reply src/app/application-dialog/application-dialog.component.ts:113]: 😎😎😎😎 agreed 😎😎😎😎

Comment (kyskiz) [reply src/app/application-dialog/application-dialog.component.ts:114]: Bindings after forEach loop are no longer needed
Just make sure form data sent is valid

**Review by kyskiz — APPROVED** [review]

**Review by swoogie — APPROVED** [review]
lgtm

**Pull request merged**